### DISCUSSION TITLE
getIsland method to IslandDisbandEvent

### DISCUSSION THREAD

**Discussion by hamza-cskn** [BG-Software-LLC/SuperiorSkyblock2#320] [Implemented Feature Requests] [answered]
i wanna a small request.

please add getIsland() method to:
https://github.com/BG-Software-LLC/SuperiorSkyblock2/blob/master/API/src/main/java/com/bgsoftware/superiorskyblock/api/events/IslandDisbandEvent.java

thanks for interest.

**Comment by OmerBenGera** [comment, marked as answer]
Already a thing...
